src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/CustomThreadsRepresentationTest.kt

```diff
@@ -1,219 +0,0 @@
-/*
- * Lincheck
- *
- * Copyright (C) 2019 - 2024 JetBrains s.r.o.
- *
- * This Source Code Form is subject to the terms of the
- * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
- * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
-package org.jetbrains.kotlinx.lincheck_test.representation
-
-
-import org.jetbrains.kotlinx.lincheck.*
-import org.jetbrains.kotlinx.lincheck.scenario
-import org.jetbrains.kotlinx.lincheck.execution.*
-import org.jetbrains.kotlinx.lincheck.strategy.*
-import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.*
-import org.jetbrains.kotlinx.lincheck.transformation.*
-import org.jetbrains.kotlinx.lincheck.util.*
-import org.jetbrains.kotlinx.lincheck.verifier.Verifier
-import org.jetbrains.kotlinx.lincheck_test.gpmc.*
-import org.jetbrains.kotlinx.lincheck_test.util.*
-import java.util.concurrent.atomic.AtomicIntegerFieldUpdater
-import kotlin.concurrent.thread
-import kotlin.reflect.KClass
-import kotlin.reflect.KFunction
-import org.junit.Test
-import java.util.concurrent.ConcurrentLinkedDeque
-
-class CustomThreadsRepresentationTest {
```

Comment:
Should this test be restored?

Reply:
It is restored under the name `CustomThreadsRunConcurrentRepresentationTest`.